src/state/projects.js

```diff
@@ -130,6 +130,12 @@ const deleteProjectUser = async (project, user) => {
   return user;
 };
 
+const leaveProject = async (project, user) => {
+  await ProjectService.leaveProject(project);
+  state.projectUsers = state.projectUsers.filter(u => u.id !== user.id);
```

Comment:
Is it useful to filter the project users as you're not a member of this project anymore and can't open this project?